app/client/components/forms/BasicDescription.tsx

```diff
@@ -0,0 +1,25 @@
+import { Description } from "@headlessui/react";
+import { FieldHandler } from "@reactivated";
+import parse from "html-react-parser";
+import React from "react";
+
+interface BasicDescriptionProps {
+  field: FieldHandler;
+}
+
+export const BasicDescription = ({ field }: BasicDescriptionProps) => {
+  if (field.help_text || field.error) {
```

Comment:
Return early pattern works better here
```suggestion
  if (!field.help_text && !field.error) {
      return <></>;
  }
```

Reply:
Done